### ISSUE TITLE
No code updates?

### ISSUE THREAD

**Issue by drew887** [droidmjt/Droidsound#194]
You seem to only be updating the README with new download links, but never updating the actual code here on github. I understand that you have this under no license, but if you don't want people to see the code then why host it on github?

**Comment by droidmjt** [comment]
you are absolutely correct, problem solved

**Comment by droidmjt** [comment]
does this bother you somehow? do you have some nice features ready to be merged to the code? :)

**Comment by drew887** [comment]
After seeing that you don't have any Bluetooth devices in #189 and confirming that it doesn't pause I thought I'd take a look, especially since on your io page you're asking for development help. 

After forking though I realised the code was quite old; so I figured I'd ask to see if it was because of your agreement with the original developer or if it was simply that you wanted to keep it private, but still wanted bug report functionality. 

Sorry, I didn't mean to come off sounding rather brash.
